includes/payment-methods/class-upe-payment-gateway.php

```diff
@@ -1065,21 +1061,6 @@ public function clear_upe_appearance_transient() {
 		delete_transient( self::WC_BLOCKS_UPE_APPEARANCE_TRANSIENT );
 	}
 
-	/**
-	 * Sets the title on checkout correctly before the title is displayed.
-	 *
-	 * @param string $title The title of the gateway being filtered.
-	 * @param string $id    The id of the gateway being filtered.
-	 *
-	 * @return string Filtered gateway title.
-	 */
-	public function maybe_filter_gateway_title( $title, $id ) {
-		if ( self::GATEWAY_ID === $id && $this->title === $title ) {
-			$title = $this->method_title;
-		}
-		return $title;
-	}
```

Comment:
Deleted this because we no longer need to change title based on payment methods. https://github.com/Automattic/woocommerce-payments/pull/3968/files#r941784012